### ISSUE TITLE
Watergate Out of Memory Crash

### ISSUE THREAD

**Issue by Jgr9** [ValveSoftware/Source-1-Games#2538]
I'm not quite sure if this started with Jungle Inferno or with the somewhat recent update to Watergate (though... it might be earlier than those, but I don't think so).  Watergate now crashes my client in the middle of a match with an Out of Memory error in Casual Matchmaking.  It usually doesn't take long, but it's not instant.  It didn't do that in the past.  Verifying the game doesn't fix it.  Not sure if it requires a fuller server.

Windows 10 64 bit
i7 2600K
16 GB RAM
GTX 770 4GB - Driver 391.01

**Comment by CommodoreKong** [comment]
This happens to me as well, when it does the console fills up with the following error: 
Failed to create decoder for MP3 [ vo\watergate\pickup_beer.mp3 ]
I've tried verifying TF2 game files but that doesn't fix the issue.

**Comment by Jgr9** [comment]
Yeah, I've tried seeing if I noticed anything consistent in the console, but there's always a mess of red at the start of loading a map (not uncommon for some reason) and then occasional ones like that while playing, but the file isn't the same.

I just tried deleting my sound.cache as mentioned here: and I had the first match on Watergate in a while that didn't crash.  But the server never filled completely.  The fix says it's temporary (or might be), so I don't know what's going wrong.

https://github.com/ValveSoftware/Source-1-Games/issues/2187
